scripts/verify_tests_on_master.bat

```diff
@@ -1,5 +1,21 @@
 @ECHO ON
 
+:: Default values.
+SET SHARDS=1
+SET SHARD_INDEX=0
+
+REM Parse the arguments for variables.
+:parse
+IF "%1"=="" GOTO endparse
+IF "%1"=="--shards" (
+    SET SHARDS=%2)
```

Comment:
nit: can the closing ) be in the next line for slightly better readibility?

Reply:
Done